action.yml

```
@@ -15,6 +15,9 @@ inputs:
     oidc-audience:
         description: "By default, this is the URL of the GitHub repository owner, such as the organization that owns the repository."
         required: false
+outputs:
+    token:
+        description: ""
```

please suggest an appropriate text here with:
````
```suggestion
        description: "a better description..."
```
````